src_c/image.c

```diff
@@ -753,6 +753,87 @@ image_tostring(PyObject *self, PyObject *arg)
         }
         pgSurface_Unlock(surfobj);
     }
+    else if (!strcmp(format, "BGRA")) {
+        hascolorkey = 0;
```

Comment:
Such a suspicious line of code, but you're doing it right, you're copying the behavior of the other ones. This is not a request for a code change or anything, this is ~~a cry for help~~ just a comment.